From a405237e9051c34580c9915813b307bc8f07310e Mon Sep 17 00:00:00 2001
From: Tyler South <tsouth2@gmail.com>
Date: Thu, 1 Oct 2026 20:06:20 -0400
Subject: [PATCH] Refresh the guest package lock

clang: 22.1.8-1 -> 23.1.1-1
compiler-rt: 22.1.8-1 -> 23.1.1-1
dua-cli: 2.45.0-1 -> 2.45.1-1
gdb: 17.2-1 -> 18.1-1
gdb-common: 17.2-1 -> 18.1-1
git: 2.55.0-1 -> 2.56.0-1
harfbuzz: 14.5.0-1 -> 14.5.1-1
libgsf: 1.14.58-1 -> 1.14.60-1
libwireplumber: 0.5.17-2 -> 0.5.18-1
linux: 7.2.7.arch1-1 -> 7.2.8.arch1-1
linux-headers: 7.2.7.arch1-1 -> 7.2.8.arch1-1
llvm-libs: 22.1.8-2 -> 23.1.1-1
mesa: 1:26.2.3-1 -> 1:26.2.3-2
noto-fonts: 1:2026.09.01-1 -> 1:2026.10.01-1
openssl: 3.6.4-1 -> 3.6.5-1
pam: 1.7.2-2 -> 1.7.3-1
pcre2: 10.48-1 -> 10.49-1
tzdata: 2026d-1 -> 2026e-1
vapoursynth: 79-1 -> 80-1
vulkan-mesa-implicit-layers: 1:26.2.3-1 -> 1:26.2.3-2
vulkan-swrast: 1:26.2.3-1 -> 1:26.2.3-2
vulkan-virtio: 1:26.2.3-1 -> 1:26.2.3-2
wireplumber: 0.5.17-2 -> 0.5.18-1
---
 guest/packages.lock.json | 46 ++++++++++++++++++++--------------------
 1 file changed, 23 insertions(+), 23 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index d0199e2..8257372 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -40,8 +40,8 @@
     "cairo": "1.18.6-1",
     "cfitsio": "1:4.7.0-1",
     "chromium": "153.0.8010.52-1",
-    "clang": "22.1.8-1",
-    "compiler-rt": "22.1.8-1",
+    "clang": "23.1.1-1",
+    "compiler-rt": "23.1.1-1",
     "coreutils": "9.12-2",
     "cpio": "2.15-3",
     "cpptrace": "1.0.4-2",
@@ -61,7 +61,7 @@
     "diffutils": "3.12-2",
     "dkms": "3.4.3-2",
     "double-conversion": "3.4.0-1",
-    "dua-cli": "2.45.0-1",
+    "dua-cli": "2.45.1-1",
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
     "elfutils": "0.196-1",
@@ -97,15 +97,15 @@
     "gcc-libs": "16.2.1+r23+gd564253eb6c8-1",
     "gcr": "3.41.2-2",
     "gcr-4": "4.4.1-1",
-    "gdb": "17.2-1",
-    "gdb-common": "17.2-1",
+    "gdb": "18.1-1",
+    "gdb-common": "18.1-1",
     "gdbm": "1.26-2",
     "gdk-pixbuf2": "2.44.7-1",
     "gettext": "1.0-2",
     "gexiv2": "0.16.2-2",
     "gexiv2-common": "0.16.2-2",
     "giflib": "6.1.3-2",
-    "git": "2.55.0-1",
+    "git": "2.56.0-1",
     "glib-networking": "1:2.80.1-1",
     "glib2": "2.88.3-1",
     "glibc": "2.44+r50+g1848099f063e-1",
@@ -151,7 +151,7 @@
     "gupnp-igd": "1.6.0-2",
     "gvfs": "1.60.3-3",
     "gzip": "1.15-1",
-    "harfbuzz": "14.5.0-1",
+    "harfbuzz": "14.5.1-1",
     "hdf5": "2.2.0-1",
     "herdr": "0.8.2-1",
     "hicolor-icon-theme": "0.18-1",
@@ -266,7 +266,7 @@
     "libglvnd": "1.7.0-3",
     "libgomp": "16.2.1+r23+gd564253eb6c8-1",
     "libgpg-error": "1.61-1",
-    "libgsf": "1.14.58-1",
+    "libgsf": "1.14.60-1",
     "libgudev": "238-3",
     "libgxps": "0.3.2-5",
     "libheif": "1.23.5-1",
@@ -382,7 +382,7 @@
     "libvterm": "0.3.3-2",
     "libwacom": "2.20.0-1",
     "libwebp": "1.6.0-2",
-    "libwireplumber": "0.5.17-2",
+    "libwireplumber": "0.5.18-1",
     "libx11": "1.8.13-1",
     "libxau": "1.0.12-1",
     "libxcb": "1.17.0-1",
@@ -418,11 +418,11 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.7.arch1-1",
+    "linux": "7.2.8.arch1-1",
     "linux-api-headers": "7.2-1",
-    "linux-headers": "7.2.7.arch1-1",
+    "linux-headers": "7.2.8.arch1-1",
     "llhttp": "9.3.1-1",
-    "llvm-libs": "22.1.8-2",
+    "llvm-libs": "23.1.1-1",
     "lm_sensors": "1:3.6.2-1",
     "lmdb": "0.9.35-1",
     "localsearch": "3.11.2-1",
@@ -438,7 +438,7 @@
     "man-db": "2.13.1-2",
     "md4c": "0.5.3-1",
     "mdadm": "4.6-2",
-    "mesa": "1:26.2.3-1",
+    "mesa": "1:26.2.3-2",
     "minizip": "1:1.3.2-3",
     "mkinitcpio": "42.1-1",
     "mkinitcpio-busybox": "1.36.1-1",
@@ -457,7 +457,7 @@
     "nettle": "4.0-1",
     "networkmanager": "1.58.1-1",
     "nftables": "1:1.1.7-3",
-    "noto-fonts": "1:2026.09.01-1",
+    "noto-fonts": "1:2026.10.01-1",
     "noto-fonts-cjk": "20240730-1",
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
@@ -479,7 +479,7 @@
     "openjpeg2": "2.5.4-1",
     "openjph": "0.32.0-1",
     "openssh": "10.5p1-1",
-    "openssl": "3.6.4-1",
+    "openssl": "3.6.5-1",
     "opus": "1.6.1-1",
     "orc": "0.4.44-1",
     "osinfo-db": "20260812-1",
@@ -488,7 +488,7 @@
     "pacman-contrib": "1.13.1-1",
     "pacman-mirrorlist": "20260610-1",
     "pahole": "1:1.32-1",
-    "pam": "1.7.2-2",
+    "pam": "1.7.3-1",
     "pambase": "20260616-1",
     "pamixer": "1.6-4",
     "pango": "1:1.58.2-1",
@@ -496,7 +496,7 @@
     "patch": "2.8-1",
     "pciutils": "3.15.0-1",
     "pcre": "8.45-5",
-    "pcre2": "10.48-1",
+    "pcre2": "10.49-1",
     "pcsclite": "2.5.2-1",
     "perl": "5.42.2-2",
     "perl-error": "0.17030-3",
@@ -594,7 +594,7 @@
     "ttf-jetbrains-mono-nerd": "3.5.1-2",
     "ttf-liberation": "2.1.5-2",
     "ttfx": "0.3.2-1",
-    "tzdata": "2026d-1",
+    "tzdata": "2026e-1",
     "uchardet": "0.0.8-4",
     "udiskie": "2.7.0-2",
     "udisks2": "2.11.2-1",
@@ -606,19 +606,19 @@
     "uwsm": "0.27.0-1",
     "v4l-utils": "1.32.0-2",
     "v4l2loopback-dkms": "0.15.4-2",
-    "vapoursynth": "79-1",
+    "vapoursynth": "80-1",
     "vid.stab": "1.1.2-1",
     "vmaf": "3.2.1-1",
     "volume_key": "0.3.12-12",
     "vulkan-icd-loader": "1.4.357.0-1",
-    "vulkan-mesa-implicit-layers": "1:26.2.3-1",
-    "vulkan-swrast": "1:26.2.3-1",
-    "vulkan-virtio": "1:26.2.3-1",
+    "vulkan-mesa-implicit-layers": "1:26.2.3-2",
+    "vulkan-swrast": "1:26.2.3-2",
+    "vulkan-virtio": "1:26.2.3-2",
     "wayland": "1.26.0-1",
     "wayland-protocols": "1.49-1",
     "webrtc-audio-processing-1": "1.3-5",
     "which": "2.25-1",
-    "wireplumber": "0.5.17-2",
+    "wireplumber": "0.5.18-1",
     "wl-clipboard": "1:2.3.0-1",
     "woff2-font-awesome": "7.3.1-1",
     "wpa_supplicant": "2:2.12-1",
-- 
2.43.0

